src/README.md

```diff
@@ -49,3 +49,24 @@ const App = () => (
 It'll grab **Open Sans** from Google's CDN [through a CSS import](https://github.com/zopaUK/react-components/blob/master/src/components/styles/Fonts.tsx#L3-L5).
 
 You're free to use this technique or add the dependency manually to your base HTML `<head />`
+
+### Spacing
+
+```ts static
+import { Margin } from '@zopauk/react-components';
+
+// root component
+const App = () => (
+  <>
+    <Margin />
+    <Padding />
```

Comment:
I suggest making this part of [`<GlobalStyles />`](https://github.com/zopaUK/react-components/blob/master/src/components/styles/GlobalStyles.tsx) instead, in this way consumers need to do less setup 💅🏻

Reply:
@thegrinder also reply for yours I guess ☝️

Reply:
I was considering that but I wasn't sure if people would like the flexibility?

Reply:
I think this should come yes or yes when using the library so that people stick our desing system 🙏🏻

Reply:
Done 👍